webui/react/src/pages/DesignKit.tsx

```diff
@@ -202,38 +218,190 @@ const DropdownsSection: React.FC = () => {
-        <SelectFilter
-          defaultValue={1}
+        <strong>Default Select</strong>
+        <Select
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Variations</strong>
+        <strong>Select with default value</strong>
+        <Select
+          defaultValue={2}
           options={[
             { label: 'Option 1', value: 1 },
             { label: 'Option 2', value: 2 },
             { label: 'Option 3', value: 3 },
           ]}
         />
-        <strong>Disabled dropdown</strong>
-        <SelectFilter
+        <strong>Select with placement</strong>
+        {placementOptions.map((placement) => (
+          <Select
+            key={placement[1]}
+            label={placement[1]}
+            options={[
+              { label: 'Option 1', value: 1 },
+              { label: 'Option 2', value: 2 },
+              { label: 'Option 3', value: 3 },
+            ]}
+            placement={placement[0]}
+          />
+        ))}
+        <strong>Select with label</strong>
+        <Select
+          label="Select Label"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Select with placeholder</strong>
+        <Select
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          placeholder="Select Placeholder"
+        />
+        <strong>Disabled Select</strong>
+        <Select
           defaultValue="disabled"
           disabled
           options={[{ label: 'Disabled', value: 'disabled' }]}
         />
-        <hr />
+        <strong>Select with search</strong>
+        <Select
+          filterOption={handleFilter}
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          placeholder="Search"
+        />
+        <strong>Multiple Select with tags</strong>
+        <Select
+          mode="multiple"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Multiple Select with tags disabled</strong>
+        <Select
+          disableTags
+          mode="multiple"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          value={multiSelectValues}
+          onChange={(value) => setMultiSelectValues(value)}
+        />
+        <strong>Select with tags and default search</strong>
+        <Select
```

Comment:
question: currently, we cannot set width, but i think select componet should accept `width` prop for better UI.

<img width="249" alt="image" src="https://user-images.githubusercontent.com/109113616/222558582-e32aabdd-a9e1-4c6e-978f-2e807e0e9ccc.png">

Reply:
In reality these should all either be setting a placeholder or have a default value. I don't think there are any dropdowns in the app that don't have at least one of those properties (or are in a form).

Reply:
Thats true, but if selected value is longer or shorter than placeholder or previous selected value, the width changes.
i think width prop (`width?: string | number`) would be helpful.

Reply:
example: 

https://user-images.githubusercontent.com/109113616/222571674-103214ca-8d73-4abc-9523-1bb475bc66f0.mov

Reply:
Component variations should be driven by _usage_. The fact that the width is changing isn't necessarily a problem. Also we could potentially just make the dropdown the width of the _widest item_ or just a general fixed width (not a customizable one).

I do think the changing width is an issue in the case where the dropdowns are displayed inline (which does happen in the app). Although the cases where we do this in latest-master actually do change width already. In my final design review I'll take a look at these cases to identify the correct sizing behavior.